### PR TITLE
UN-676: Record Article new label

### PR DESCRIPTION
**Pull request by jamesbiggs**
### Please do not merge this PR when accepted

Ticket URL: https://national-archives.atlassian.net/browse/UN-676

## About these changes

Added `NewLabelMixin` to `RecordArticlePage`.

## How to check these changes

Edit a `RecordArticlePage` and check the "new label" attribute is there. 

## Before assigning to reviewer, please make sure you have

- [x] Checked things thoroughly before handing over to reviewer.
- [x] Checked PR title starts with ticket number as per project conventions to help us keep track of changes.
- [x] Ensured that PR includes only commits relevant to the ticket.
- [x] Waited for all CI jobs to pass before requesting a review.
- [x] Added/updated tests and documentation where relevant.

## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)

### CODE REVIEW

**Comment by TNA-Allan** [comment]
> Note to reviewer
Please do not merge this PR when accepted

What the reason for this?

And can it me moved below (as well) 
>  Merging PR guidance

**Please do not merge this PR when accepted**

**Comment by jamesbiggs** [comment]
> > Note to reviewer
> > Please do not merge this PR when accepted
> 
> What the reason for this?
> 
> And can it me moved below (as well)
> 
> > Merging PR guidance
> 
> **Please do not merge this PR when accepted**

Hi Allan, just putting this here as it was discussed on slack:

This was done as I have migrations that are overlapping in different PRs :)

**Review by TNA-Allan — APPROVED** [review]
@jamesbiggs Thanks for your clarifications. Looks good. Approving. Please merge as required

**Pull request merged**
